site/docs/hosting/vps.md

```diff
@@ -2,9 +2,84 @@
 
 A virtual private server, mostly known as VPS, is a virtual machine running in the cloud with its users having the full control of its system.
 
-In this guide, you'll learn about various methods of running your bot in a VPS, keeping it online 24/7, making it run automatically when your VPS boots and restart on crashes.
+## Preparing Your Code
 
-## systemd
+To get started, you can use the bot shown in the [example of using a session](../plugins/session.md#example-usage).
+
+In the following, we will assume that the bot runs on Node.js
+Why only Node.js and not Deno or is it better for both runtimes?
+The answer is simple: the Deno pipeline would be very simple.
+A Node.js pipeline is complicated by the additional task of building and setting up communication between the two tasks.
+So a CI/CD pipeline for Node.js would be a better example.
```

Comment:
This logic is gold, but we're also making it easier to use Node now because there are no Deno docs. If it's so simple, couldn't we just provide both tutorials?

Reply:
We can do it the same way we do it in the getting started guide